flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientITCase.java

```diff
@@ -83,15 +83,20 @@
 
     @Parameterized.Parameters(name = "{0}")
     public static Object[] parameters() throws Exception {
-        String first = "sql/table.q";
-        URL url = CliClientITCase.class.getResource("/" + first);
-        File firstFile = Paths.get(url.toURI()).toFile();
-        final int commonPrefixLength = firstFile.getAbsolutePath().length() - first.length();
-        File dir = firstFile.getParentFile();
+        String sqlDir = "sql";
```

Comment:
Why change this? I think we don't need to change this and all new SQL scripts can be discovered.